Description: ensure that sections in manpages are consistent (lintian errors)
Author: Rolf Leggewie <foss@rolf.leggewie.biz>
Forwarded: No
Last-Update: 2010-09-12

Index: isdnutils/xisdnload/xisdnload.man
===================================================================
--- isdnutils.orig/xisdnload/xisdnload.man	2010-09-12 17:40:37.000000000 +0200
+++ isdnutils/xisdnload/xisdnload.man	2010-09-12 18:12:16.000000000 +0200
@@ -1,5 +1,5 @@
 .\" based on: $XConsortium: xload.man,v 1.29 94/04/17 20:43:44 matt Exp $
-.TH XISDNLOAD 1 "Release 6" "X Version 11"
+.TH XISDNLOAD 1x "Release 6" "X Version 11"
 .SH NAME
 xisdnload \- ISDN load average display for X
 .SH SYNOPSIS
Index: isdnutils/imontty/imontty.8.in
===================================================================
--- isdnutils.orig/imontty/imontty.8.in	2010-09-12 17:38:28.000000000 +0200
+++ isdnutils/imontty/imontty.8.in	2010-09-12 18:13:12.000000000 +0200
@@ -6,7 +6,7 @@
 .\" groff -man -Tascii imontty.8 for ASCII output, or
 .\" groff -man -Tps imontty.8 for PostScript output
 .\"
-.TH IMONTTY 8 "@MANDATE@" isdn4k-utils-@I4LVERSION@ "Linux System Administration"
+.TH IMONTTY 1 "@MANDATE@" isdn4k-utils-@I4LVERSION@ "Linux System Administration"
 .SH NAME
 imontty \- display status of all ISDN lines
 .SH SYNOPSIS
Index: isdnutils/imon/imon.man.in
===================================================================
--- isdnutils.orig/imon/imon.man.in	2010-09-12 17:35:10.000000000 +0200
+++ isdnutils/imon/imon.man.in	2010-09-12 18:13:48.000000000 +0200
@@ -6,7 +6,7 @@
 .\" groff -man -Tascii imon.8 for ASCII output, or
 .\" groff -man -Tps imon.8 for PostScript output
 .\"
-.TH IMON 8 "@MANDATE@" isdn4k-utils-@I4LVERSION@ "Linux System Administration"
+.TH IMON 1 "@MANDATE@" isdn4k-utils-@I4LVERSION@ "Linux System Administration"
 .SH NAME
 imon \- watch ISDN activities
 .SH SYNOPSIS
